app/importers/envirotech/analysis_link_data.rb

```diff
@@ -42,7 +42,7 @@ def process_article_info(article_hash)
 
       article[:source] = model_class.source[:code]
 
-      article[:id] = Utils.generate_id(article, %i(source_id source))
+      article[:id] = article[:source_id]
```

Comment:
Yeah, since each source has it's own index, we can safely use the "source_id" as the document id. It makes `get`ting a document much easier over in the ProviderSolutions importer, which I've changed in this PR.